src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -79,17 +76,18 @@ export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
 
   return (
     <>
-      <div className={containerClasses} onClick={showEventModal}>
-        <div className="ScheduleEvent__info">
-          <div className="ScheduleEvent__title">{event.name}</div>
-          <div className="ScheduleEvent__host">by {event.host}</div>
-        </div>
-
-        <div className="ScheduleEvent__bookmark" onClick={bookmarkEvent}>
-          <FontAwesomeIcon
-            icon={event.isSaved ? solidBookmark : regularBookmark}
-          />
+      <div className={containerClasses}>
```

Comment:
This was changed to prevent the event modal from popping up whenever a bookmark was clicked. This appears to be a bug in the current implementation, which is resolved by splitting the class from the onclick activity